kythe/rust/indexer/src/indexer/analyzers.rs

```diff
@@ -97,9 +97,11 @@ impl<'a> UnitAnalyzer<'a> {
         for required_input in unit.get_required_input() {
             let analysis_vname = required_input.get_v_name();
             let storage_vname: VName = analysis_to_storage_vname(analysis_vname);
-            let path = storage_vname.get_path().to_owned();
+            let info = required_input.get_info();
+            let path = info.get_path().to_owned();
+            let digest = info.get_digest().to_owned();
```

Comment:
This still won't work for generated files?

Reply:
Okay I fixed a small bug and now I have confirmed that this code works for generated files. The last question is whether the path in the vnames for the references should have the compiler file path or the processed file path?